pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -127,6 +127,14 @@ type ClusterStatus struct {
 	// NodeSummary represents the summary of nodes status in the member cluster.
 	// +optional
 	NodeSummary NodeSummary `json:"nodeSummary,omitempty"`
+
+	// Nodes represents the numbers of all nodes and ready nodes in the member cluster.
+	// +optional
+	Nodes Nodes `json:"nodes,omitempty"`
+
+	// Resources represents the allocatable, allocating and allocated resources in the member cluster.
+	// +optional
+	Resources Resources `json:"resource,omitempty"`
```

Comment:
`Resource` might be an uncountable word, how about change `Resources` --> `Resource`?

Reply:
We cannot. `Resource` has been used according to